### DISCUSSION TITLE
Table of Content not showing h1 tags #

### DISCUSSION THREAD

**Discussion by schadokar** [jpanther/congo#217] [Q&A] [answered]
The New table of content is amazing and crisp.
One thing I observed is, that it shows content from `##` instead of `#`.
My blogs always start with `#`.
What do I have to do to make this work?

**Comment by jpanther** [comment, marked as answer]
The tables of contents follow standard Hugo [TOC configuration parameters](https://gohugo.io/getting-started/configuration-markup/#table-of-contents). By default, these display headings at levels 2 through 4. If you want to change this behaviour, adjust the values in the `config/_default/markup.toml` file. In your case you'll need to set `startLevel` to 1.

```toml
[tableOfContents]
  startLevel = 1
  endLevel = 4
```
